SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialTraceEventBuilder.cpp

```diff
@@ -116,6 +116,13 @@ FSpatialTraceEvent FSpatialTraceEventBuilder::CreateSendRPC(const UObject* Objec
 		.GetEvent();
 }
 
+FSpatialTraceEvent FSpatialTraceEventBuilder::CreateRPCLinearTraceEvent(const EventTraceUniqueId& LinearTraceId)
+{
+	return FSpatialTraceEventBuilder(GDK_EVENT_NAMESPACE "rpc_linear_trace")
```

Comment:
Soft suggestion that I'd prefer us to, by convention, have event types describe what's happening (send rpc, receive rpc) rather than what use case they exist to solve.